controller/organization.go

```diff
@@ -0,0 +1,161 @@
+package controller
+
+import (
+	"github.com/fabric8-services/fabric8-auth/account"
+	"github.com/fabric8-services/fabric8-auth/app"
+	"github.com/fabric8-services/fabric8-auth/application"
+	"github.com/fabric8-services/fabric8-auth/authorization/resource"
+	"github.com/fabric8-services/fabric8-auth/authorization/role"
+	"github.com/fabric8-services/fabric8-auth/errors"
+	"github.com/fabric8-services/fabric8-auth/jsonapi"
+	"github.com/fabric8-services/fabric8-auth/log"
+	"github.com/fabric8-services/fabric8-auth/login"
+	"github.com/fabric8-services/fabric8-auth/token"
+	"github.com/goadesign/goa"
+	"github.com/satori/go.uuid"
+	"strings"
+)
+
+const OrganizationOwnerRole = "owner"
+
+// OrganizationController implements the organization resource.
+type OrganizationController struct {
+	*goa.Controller
+	db           application.DB
+	TokenManager token.Manager
+}
+
+// NewOrganizationController creates an organization controller.
+func NewOrganizationController(service *goa.Service, db application.DB) *OrganizationController {
+	return &OrganizationController{Controller: service.NewController("OrganizationController"), db: db}
+}
+
+// Create runs the create action.
+func (c *OrganizationController) Create(ctx *app.CreateOrganizationContext) error {
+	currentUser, err := login.ContextIdentity(ctx)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
+	}
+
+	if len(strings.TrimSpace(*ctx.Payload.Name)) == 0 {
```

Comment:
might want to check if `ctx.Payload.Name` != nil as well?

Reply:
Good idea, done.